### ISSUE TITLE
experimental setup

### ISSUE THREAD

**Issue by cszmli** [truthless11/GDPL#2]
Hi, I rerun the experimental setup based on the parameters you updated but only got 0.566 with respect to the success rate. I increased the training epoch to 200 and the success rate was still below 0.7. Much appreciate if you can go through the setup again.
<img width="747" alt="Screenshot 2019-09-27 at 09 12 36" src="https://user-images.githubusercontent.com/18543171/65749728-fbad4200-e106-11e9-82ab-0905563a9f3d.png">

_Originally posted by @cszmli in https://github.com/truthless11/GDPL/issues/1#issuecomment-535821983_

**Comment by truthless11** [comment]
hi, @cszmli 
I've rerun the code and found nothing wrong with the training process and its performance. I update the best model that I got in the zip file and the result is shown below.  

> INFO:root:Load processed data file
DEBUG:root:test
INFO:root:Loading goal model is done
INFO:root:<<reward estimator>> loaded checkpoint from file: data/agenda/best_estimator.mdl
INFO:root:<<dialog policy>> loaded checkpoint from file: data/agenda/best_ppo.val.mdl
INFO:root:<<dialog policy>> loaded checkpoint from file: data/agenda/best_ppo.pol.mdl
INFO:root:reward -85.81368823640744
INFO:root:turn 7.638
INFO:root:match 0.8598425196850393
INFO:root:inform rec 0.9579639691163855, F1 0.9176825092453088
INFO:root:success 0.882

After the pretraining, you should get nearly 60% success rate, and here are the parameters that I used for training the policy

> DEBUG:root:Namespace(anneal=5000, batchsz=32, batchsz_traj=1024, clip=0.03, config='multiwoz', data_dir='data', epoch=16, epsilon=0.2, gamma=0.99, load='model_agenda_pre/best', load_user='model_agenda/best', log_dir='log', lr_auto=0.001, lr_irl=0.0001, lr_rl=0.0001, lr_simu=0.001, pretrain=False, print_per_batch=400, process=16, save_dir='model_agenda', save_per_epoch=1, simulator='agenda', tau=0.95, test=False, update_round=5)

May this help you.

**Comment by cszmli** [comment]
problem solved. I got 0.835 with the parameters below.
`Namespace(anneal=5000, batchsz=32, batchsz_traj=1024, clip=0.02, config='multiwoz', data_dir='data', epoch=16, epsilon=0.2, gamma=0.99, load='model_agenda_pre/best', load_user='model_agenda_irl/best', log_dir='log', lr_irl=0.0001, lr_rl=0.0001, lr_simu=0.001, pretrain=False, print_per_batch=400, process=16, save_dir='model_agenda_irl', save_per_epoch=5, simulator='agenda', tau=0.95, test=False, update_round=5)`. Thanks for your reply. By the way, it looks like the pretrained model is getting the same performance with other baselines or even beating ACER in the paper.

**Comment by Jeiyoon** [comment]
Hi, I have same problem with cszmli and i cant find your zip file what you uploaded. Could you let me know where your model zipfile is?

**Comment by Jeiyoon** [comment]
and could you re-upload your codes? i have no parameters like lr_auto=0.001. thank you so much

**Comment by truthless11** [comment]
@Jeiyoon Just ignore the hyperparameter lr_auto, it is useless and I remove it from the source code. You can simply run the script fetch_data.sh to get the trained model and data.

**Comment by Jeiyoon** [comment]
hi, i read your paper GDPL(https://arxiv.org/abs/1908.10719) and ran your code (https://github.com/truthless11/GDPL) but even i tried many times, i cant get result what your team get in your paper

I also follow all your instructions written on README and closed issue and put right parameters into codes

and i got more than 60% success rate during the pretraining process (62%)

I dont know what is the problem

heres parameters during train process

Namespace(anneal=5000, batchsz=32, batchsz_traj=1024, clip=0.03, config='multiwoz', data_dir='data', epoch=16, epsilon=0.2, gamma=0.99, load='best0619/best', load_user='model_agenda/best', log_dir='log', lr_irl=0.0001, lr_rl=0.0001, lr_simu=0.001, pretrain=False, print_per_batch=400, process=16, save_dir='model_agenda', save_per_epoch=1, simulator='agenda', tau=0.95, test=False, update_round=5)

and i got 20% to 50% success rate even i tried so many times

so would you mind if you let me know more details for training like pretrain, train and test parameters and procedures

thank you so much

**Comment by Jeiyoon** [comment]
heres my training procedures. all parameters are same 

1) load pretrained models and save trained models

'--load', type=str, default='best0619/best' (pretrained model directory)
'--save_dir', type=str, default='model_agenda'

2) load trained models for test

'--load', type=str, default='model_agenda/best' (trained model directory)
